src/knex-tables.d.ts

```diff
@@ -148,10 +149,75 @@ declare module "knex/types/tables" {
     id: number;
     onerep_profile_id: number;
     onerep_scan_id: number;
-    onerep_scan_results: ScanResult;
+    onerep_scan_reason: "manual" | "initial" | "monitoring";
```

Comment:
Can this be `Scan["reason"]` (from `onerep.ts`)

Reply:
Oh nice, yes it should be: dc2c0cf43